config.toml

```diff
@@ -219,3 +219,8 @@ enable = false
 #    identifier = "openpnp"
 #    url = "openpnp"
 #    weight = 20
+
+[module]
+[[module.mounts]]
+        source = 'assets'
+        target = 'assets'
```

Comment:
@theacodes FYI, this configuration line was needed to let all of the docs pages grab images from `assets/images`. Up until now, all the images had been specific to each step so they'd all lived in each step's specific `images` folder.

Reply:
thanks!